pkg/daemon/ceph/client/filesystem.go

```diff
@@ -309,6 +309,43 @@ func FailMDS(context *clusterd.Context, clusterName string, gid int) error {
 	return nil
 }
 
+// DownFilesystem marks the filesystem as down and the MDS' as failed
+func DownFilesystem(context *clusterd.Context, cephVersion cephver.CephVersion, clusterName, filesystemName string) error {
```

Comment:
Functions should be single purpose here. This function is doing two things, marking the fs as down and also marking the mdses as failed. There are already client commands for both of these things individually, and we should keep those in case they are needed separately in the future.

Reply:
Done